addons/project/static/tests/tours/project_tags_filter_tour_tests.js

```diff
@@ -0,0 +1,95 @@
+/** @odoo-module **/
+
+import tour from 'web_tour.tour';
+
+function fail(errorMessage) {
+    tour._consume_tour(tour.running_tour, errorMessage);
+}
+
+function assert(current, expected, info) {
+    if (current !== expected) {
+        fail(info + ': "' + current + '" instead of "' + expected + '".');
+    }
+}
+
+tour.register('project_tags_filter_tour', {
+    test: true,
+    url: '/web',
+},
+[tour.stepUtils.showAppsMenuItem(), 
+{
+    trigger: '.o_app[data-menu-xmlid="project.menu_main_pm"]',
+},
+{
+    trigger: '.o_favorite_menu button:has(i.fa-star)',
+    content: 'click on the favorite menu',
+},
+{
+    trigger: '.o_favorite_menu .dropdown-item span:contains("pink tag filter")',
+},
+{
+    trigger: '.o_kanban_renderer',
+    content: 'check that the tag displays the correct records',
+    run: async function() {
+        await new Promise((r) => setTimeout(r, 1000));
+        var records = $('.o_kanban_record.oe_kanban_global_click');
```

Comment:
Avoid using `var`, use `cont` for constants or `let` for variables instead

Reply:
Yes mb